components/Checklist.js

```diff
@@ -0,0 +1,12 @@
+export default function Checklist({menuItems}, {menuName}) {
+    return (
+        <div>
+            <button type="button" class="checklist">{menuName}</button>
+            {menuItems.map((m) => (
+            <div>
+            <label>{m}</label>
```

Comment:
For accessibility, the <label> should wrap the input like this (this allows screen readers to know what the label is labeling, and also it allows users to click the label to focus the checkbox. So it is good for accessibility and usability):

```
<label>
    {m}
    <input type="checkbox" />
</label>
```